lib/lhm/throttler/slave_lag.rb

```diff
@@ -51,14 +52,19 @@ def slaves
 
       def get_slaves
         slaves = []
-        slave_hosts = master_slave_hosts
-        while slave_hosts.any? do
-          host = slave_hosts.pop
-          slave = Slave.new(host, @get_config)
-          if slaves.map(&:host).exclude?(host) && slave.connection
-            slaves << slave
-            slave_hosts.concat(slave.slave_hosts)
+        if @check_only.nil?
+          slave_hosts = master_slave_hosts
+          while slave_hosts.any? do
+            host = slave_hosts.pop
+            slave = Slave.new(host, @get_config)
+            if slaves.map(&:host).exclude?(host) && slave.connection
+              slaves << slave
+              slave_hosts.concat(slave.slave_hosts)
+            end
           end
+        else
+          slave_config = @check_only.call
```

Comment:
Is it worth adding a check (and a corresponding test...) here in case the parameter passed in is not something that responds to "call"?